app/controllers/miq_policy_controller.rb

```diff
@@ -560,7 +560,7 @@ def replace_right_cell(options = {})
         end
 
       presenter.update(:main_div, r[:partial => partial_name])
-      right_cell_text = _("All %{models}") % {:models => model}
```

Comment:
@GregP i think we should just change https://github.com/GregP/manageiq-ui-classic/blob/28ff2b3b5628e99b8aecfb1c120151eab63b98e4/app/controllers/miq_policy_controller.rb#L553-L554 to have pluralized values, and skip ui_lookup call